bot.py

```
@@ -98,18 +99,18 @@ async def info(message: Message, bot: Bot):
-        "Выбрать имя — Полная функциональность,\n"
+        "Выбрать имя — Полная функциональность.\n"
         "Кубик — полная функциональность.\n"
         "Отзыв🆕: Теперь вы можете оставить отзыв про бота!\n"
-        "*Память*🧠: *Публичный предпросмотр*\n"
+        "*Память*🧠: *Полная функциональность*.\n"
         "*Расшифровка голосовых сообщений в текст*:\n"
         "Просто отправьте или перешлите голосовое сообщение и я его расшифрую\n"
         "Бесплатно 10 голосовых сообщений в неделю, "
-        "далее 5 звёзд за сообщение\n\n"
+        "далее 5 звёзд за сообщение\n"
+        "Настройки > Донат: *Не даёт привилегия*: заплатите 10 звёзд чтобы поддержать разработчика\n\n"
```

Ruff (0.12.2)

String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)